.github/workflows/e2e-nm-angular-workflow.yml

```diff
@@ -44,12 +44,13 @@ jobs:
       run: |
         source scripts/e2e-setup-ci.sh nm
```

Comment:
This still setup yarn with node-modules linker, because of `nm` arg. That seems counter-intuitive with the PR goal. Is that on purpose ?

Reply:
That's on purpose, we test it with both PnP and `node-modules`